### ISSUE TITLE
Input field for ward (when pref enabled)

### ISSUE THREAD

**Issue by jmbrunskill** [msupply-foundation/open-msupply#5797]

**Comment by jmbrunskill** [comment]
Adding `needs refinement` as we need to decide how/where to display these properties...

**Comment by jmbrunskill** [comment]
@mark-prins  the `options` table is actually called `reason_options` and has `reason`s

So I think maybe we stick with the names as wards solution for now?
I'm not clear on if the wards are linked to a store or not, I feel like potentially you might have the same ward served by multiple stores, so I'm guessing we can't just make them a new type as remote data can we?

I quite like the idea of making them a new concept similar to locations? But would definitely increase the work for this feature...

**Comment by mark-prins** [comment]
yes, I like the idea of making them as a new concept but agree, let's not do it now. 
am checking - names are by default tied to stores i.e. setting up as remote data is fine. I could see the case for setting up a ward and re-using the same name across multiple hospitals in a country tho

**Comment by mark-prins** [comment]
Refinement: add a column to name of `is_ward` and populate using a translator which check the `name_tag` & `name_tag_join` tables
